src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -269,7 +269,7 @@ def _flexible_server_params(command_group):
                            help='Compute tier of the server. Accepted values: Burstable, GeneralPurpose, Memory Optimized ')
                 c.argument('sku_name', default='Standard_B1ms', options_list=['--sku-name'],
                            help='The name of the compute SKU. Follows the convention Standard_{VM name}. Examples: Standard_B1ms, Standard_E16ds_v4 ')
-                c.argument('storage_mb', default='10', options_list=['--storage-size'], type=int,
+                c.argument('storage_mb', default='32', options_list=['--storage-size'], type=int,
```

Comment:
default value change is breaking change to customer. please add in release note.

Reply:
Added it